### PR TITLE
feat(marketing): Update marketing message

### PR DESCRIPTION
**Pull request by kfrancoi**
As I'll be attending the Digital First event tomorrow, I figure it was time to update the marketing message on the Kosmio landing page.  

This is only a quick improvement, as the whole website (and marketing strategy that goes with it) will have to be redesigned.

### CODE REVIEW

**Comment by kfrancoi** [comment]
[XS(Kosmio-Landing): Quick update to Kosmio landing page](https://trello.com/c/dCpKULLU/2157-xskosmio-landing-quick-update-to-kosmio-landing-page)

File: src/index.html [hunk 1]
```diff
@@ -89,81 +89,64 @@
...
 							<div class="row uniform text-left">
 								<div class="3u 12u$(small)">
-									<span class="image bigger"><img src="images/machine-learning.svg" alt="" /></span>
+									<span class="image contain"><img src="images/levelup.svg" alt="" /></span>
 								</div>
 								<div class="9u 12u$(small)">
-									<h3 class="accent">Collect & Analyse</h3>
-									<p>Kosmio continuously tracks hundreds of thousands of news sources, blogs, scientific papers or company websites, generating millions of news every day. Loaded with text understanding capabilities in context, Kosmio collects, structures and pre-analyses the information trapped into every document it reads.</p>
+									<h3 class="accent">Detect Sales Opportunities.</h3>
+									<p>Stop wasting hours searching on Google, Newsletters and reports on the lookout for relevant information about your customers or prospect, not being sure you didn't miss anything. Kosmio is automatically doing that for you so you can gain time by acting on leads rather than searching for ones.</p>
```
Comment (fbeuvens) [inline src/index.html]: Prospect with `s`

File: src/index.html [hunk 2]
```diff
@@ -89,81 +89,64 @@
...
 							<div class="row uniform text-left">
 								<div class="3u 12u$(small)">
-									<span class="image bigger"><img src="images/hands.svg" alt="" /></span>
+									<span class="image bigger"><img src="images/competitive-analytics.svg" alt="" /></span>
 								</div>
 								<div class="9u 12u$(small)">
-									<h3 class="accent">Stay up to date, automatically</h3>
-									<p>Turn on the monitoring around the subjects that matters and let Kosmio keep you up to date with the relevant information.</p>
+									<h3 class="accent">Identify emerging trends</h3>
+									<p>Official company classification can only go so far. Activate continuous monitoring of sector or company to stay up to date with their real and evolving activity domain or stay alert about any new actors coming up in a new sector. For example, ask yourself which companies are moving toward smart-city or cyber-security.</p>
```
Comment (fbeuvens) [inline src/index.html]: `stay alerted` non?

File: src/index.html [hunk 3]
```diff
@@ -187,23 +170,71 @@ <h3 class="accent">Monitor</h3>
 				</div>
 			</section>
 
+		<!-- Two -->
+			<section id="features" class="main style3 special">
+				<div class="container">
+					<header class="major">
+						<h2>Kosmio Computational Knowledge Engine</h2>
+					</header>
+					<h3>Advanced Text Analytic solution that automatically monitors and analyzes the world’s collective intelligence on the Internet in order to help sales people in banks stay aware of the business information related to their customers that they can act on, saving time and gaining insights.</h3>
```
Comment (fbeuvens) [inline src/index.html]: Est-ce que c'est bien d'être spécifique aux banques? Est-ce que ça représente un target suffisamment large?

File: src/index.html [hunk 4]
```diff
@@ -187,23 +170,71 @@ <h3 class="accent">Monitor</h3>
...
+					<div class="row">
+						<div class="4u 12u$(small)">
+							<span class="image contain"><img src="images/machine-learning.svg" alt="" /></span>
+
+							<h3 class="accent">Artificial Intelligence & Natural Language Processing</h3>
+							<p>Kosmio uses artificial intelligence and natural language processing, in multiple languages, to identify new relevant sources, extract the right portion of the text on each web pages, structure the text with mentions of companies and topics and get rid of duplicate information, so that you don't have to.</p>
+							<!-- <p>Kosmio is based on machine learning algorithms trained to identify the relevancy of each business event.  If a classification is not correct, users can optionally send feedbacks to get less or more of certain type of news.</p> -->
+							<!-- <h3 class="accent">Landscaping</h3> -->
+							<!-- <p>Stop wasting time reading documents without added value for you and gain a better understanding of any subjects in minutes.  Kosmio start by the big picture and let the user navigate the information intuitively.</p> -->
```
Comment (fbeuvens) [inline src/index.html:198]: start avec `s`

File: src/index.html (hunk 1 above)
Comment (kfrancoi) [reply src/index.html]: 👍

**Pull request merged**